Content.Server/Atmos/EntitySystems/AtmosphereSystem.API.cs

```diff
@@ -108,7 +108,12 @@ public void InvalidateTile(EntityUid gridUid, Vector2i tile)
             RaiseLocalEvent(ref ev);
 
         // Default to a space mixture... This is a space game, after all!
-        return ev.Mixtures ?? new GasMixture?[tiles.Count];
+        ev.Mixtures ??= new GasMixture?[tiles.Count];
```

Comment:
Maybe it should just check if the event was handled here after raising it again, and then replace it to the space gas mixture without the null checks if the event wasn't handled.

Reply:
yeah that makes more sense, will do